packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -1420,15 +1419,17 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
         metricCalculator.resetCalculatedRowHeights();
       }
 
-      const { density } = this.props;
+      let densityTheme = {};
+      if (density === 'compact') {
+        densityTheme = baseTheme.density.compact;
+      }
+      if (density === 'spacious') {
```

Comment:
Minor, doesn't really matter
```suggestion
      } else if (density === 'spacious') {
```